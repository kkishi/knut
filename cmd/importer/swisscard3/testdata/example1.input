Transaktionsdatum,Beschreibung,Händler,Kartennummer,Währung,Betrag,Fremdwährung,Betrag in Fremdwährung,Debit/Kredit,Status,Händlerkategorie,Registrierte Kategorie
"21.03.2023","a","","1234","CHF","13.90","","","Belastung","Ausstehend","FAST FOOD RESTAURANTS",""
"20.03.2023","b","","1234","EUR","0.10","","","Belastung","Ausstehend","TRAVEL AGENCIES",""
"20.03.2023","c","","1234","CHF","0.70","","","Belastung","Ausstehend","DEPARTMENT STORES",""
"20.03.2023","d","","1234","CHF","5.50","","","Belastung","Ausstehend","CANDY, NUT, CONFECTIONERY STORES",""
"19.03.2023","e","","1234","CHF","51.20","","","Belastung","Gebucht","BAKERIES",""
"19.03.2023","f","","1234","CHF","9.80","","","Belastung","Gebucht","PASSENGER RAILWAYS",""
"19.03.2023","g","","1234","CHF","177.40","","","Belastung","Gebucht","EATING PLACES, RESTAURANTS",""
"18.03.2023","h","","1234","CHF","27.50","","","Belastung","Gebucht","CARD, GIFT AND NOVELTY STORES",""