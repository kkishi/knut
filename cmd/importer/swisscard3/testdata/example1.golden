2023-03-18 "h / 1234 / CARD, GIFT AND NOVELTY STORES / Belastung"
Liabilities:CreditCard Expenses:TBD                 27.5 CHF

2023-03-19 "e / 1234 / BAKERIES / Belastung"
Liabilities:CreditCard Expenses:TBD                 51.2 CHF

2023-03-19 "f / 1234 / PASSENGER RAILWAYS / Belastung"
Liabilities:CreditCard Expenses:TBD                  9.8 CHF

2023-03-19 "g / 1234 / EATING PLACES, RESTAURANTS / Belastung"
Liabilities:CreditCard Expenses:TBD                177.4 CHF

2023-03-20 "b / 1234 / TRAVEL AGENCIES / Belastung"
Liabilities:CreditCard Expenses:TBD                  0.1 EUR

2023-03-20 "c / 1234 / DEPARTMENT STORES / Belastung"
Liabilities:CreditCard Expenses:TBD                  0.7 CHF

2023-03-20 "d / 1234 / CANDY, NUT, CONFECTIONERY STORES / Belastung"
Liabilities:CreditCard Expenses:TBD                  5.5 CHF

2023-03-21 "a / 1234 / FAST FOOD RESTAURANTS / Belastung"
Liabilities:CreditCard Expenses:TBD                 13.9 CHF

